task/backend/scheduler.go

```diff
@@ -51,6 +51,9 @@ type QueuedRun struct {
 	// The Unix timestamp (seconds since January 1, 1970 UTC) that will be set when a run a manually requested
 	RequestedAt int64
 
+	// The Unix timestamp representing when this run was due to run.
+	DueAt int64
```

Comment:
Could this be a time.Time rather than int64?  The majority of this patch is conversions of time.Time to this number

Reply:
The rest of the time stamps in the QueuedRun are int64 unix time stamps. I assumed it would be better to have the model be comparable. This could be a time.Time. I don't really have a preference.